nnpdf_data/nnpdf_data/new_commondata/ATLAS_Z0_8TEV/metadata.yaml

```diff
@@ -0,0 +1,59 @@
+# Generalia
+setname: "ATLAS_Z0_8TEV"
```

Comment:
We already have `ATLAS_Z0_8TEV_LOWMASS` and `ATLAS_Z0_8TEV_HIMASS`. I think that unless this dataset supersedes both of them, it should have some other qualifier @enocera